book/standard_library.md

```diff
@@ -14,32 +18,218 @@ The standard library currently includes:
+
+- `use std/assert`: `assert` and its subcommands
+- `use std/bench`: The benchmarking command `bench`
+- `use std/dirs`: The directory stack command `dirs` and its subcommands
+- `use std/input`: The `input display` command
+- `use std/help`: An alternative version of the `help` command and its subcommands which supports completion and other features
+- `use std/iters`: Additional `iters`-prefixed iteration commands. Note: See-also alternative option #3 below.
+- `use std/log`: The `log <subcommands>` such as `log warning <msg>`
+- `use std/math`: Mathematical constants such as `$math.E`. These can also be imported without a prefix using Form #2 below.
+
+#### 2. Import all submodule commands without a prefix
+
+For certain submodules, you will want the commands from a submodule to be available in the current namespace, so that you _can_ simply access the command by name. For instance:
+
+```nu
+use std/formats *
+ls | to jsonl
+```
+
+Submodules that are normally imported with `use std/<submodule> *`:
+
+- `use std/dt *`: Additional commands for working with `date` values
+- `use std/formats *`: Additional `to` and `from` format conversions
+- `use std/math *`: The math constants without a prefix, such as `$E`. Note that the prefixed form #1 above is likely more understandable when reading and maintaining code.
+- `use std/xml *`: Additional commands for working with XML data
+
+#### 3. Importing specific subcommands
+
+As with most modules, you can choose to import only a subset of the commands. For instance, the following would import the `zip-with` command without requiring that it be called with`iter zip-with`.
+
+```nu
+use std/iter [ zip-with ]
+```
+
+#### 4. Not Recommended - `use std <submodule>` and `use std <submodule> *`
+
+While it is _possible_ to import Standard Library submodules using a space-separated form:
+
+```nu
+use std log
+use std formats *
+```
+
+This is NOT RECOMMENDED. Using this form has the same performance impact as:
+
+```nu
+use std *
+```
+
+::: caution
+The difference is subtle, but important from a performance perspective. Using `std/<submodules>` (with a slash) is far faster than `std <submodule>` (with a space):
+
+```nu
+bench -n 200 --pretty { nu -c "use std/formats *; exit" }
+# => 9ms 271µs 832ns +/- 328µs 860ns
+bench -n 200 --pretty { nu -c "use std formats *; exit" }
+# => 26ms 921µs 59ns +/- 588µs 919ns
 ```
 
-In addition, `stdlib-candidate`, found in the [nu_scripts Repository](https://github.com/nushell/nu_scripts/tree/main/stdlib-candidate/std-rfc), serves as a staging ground for new commands before they are added to the standard library.
+:::
+
+## The Standard Library Candidate Module
+
+(Also known as `std-rfc`)
+
+`stdlib-candidate`, found in the [nu_scripts Repository](https://github.com/nushell/nu_scripts/tree/main/stdlib-candidate/std-rfc), serves as a staging ground for new commands before they are added to the Standard Library.
```

Comment:
Stdlib is not only about commands. Modules can define other stuff, for example constants in the `math` module.

Reply:
Updated (and rephrased) to use the word "additions" - Thanks!